### PR TITLE
feat: add kitexcall

### PR DESCRIPTION
**Pull request by Zzhiter**
#### Check the PR title.

- [√] This PR title match the format: \<type\>(optional scope): \<description\>
- [√] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.
添加kitexcall的文档

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: content/en/docs/kitex/Tutorials/cli-tool/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: "CLI Tool"
```
Comment (felix021) [inline content/en/docs/kitex/Tutorials/cli-tool/_index.md:2]: Tool

File: content/en/docs/kitex/Tutorials/cli-tool/_index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+linkTitle: "CLI Tool"
+weight: 6
```
Comment (felix021) [inline content/en/docs/kitex/Tutorials/cli-tool/_index.md:4]: weight 6 is used by `framework-extern`. Make it 8 which is larger than option and this entry will be placed after option.

File: content/en/docs/kitex/Tutorials/cli-tool/kitexcall.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
+---
+title: "CLI tool kitexcall"
```
Comment (felix021) [inline content/en/docs/kitex/Tutorials/cli-tool/kitexcall.md:2]: kitexcall: the CLI tool for sending RPC call in JSON format

Comment (Zzhiter) [reply content/en/docs/kitex/Tutorials/cli-tool/kitexcall.md]: all fixed and checked

**Pull request merged**